### DISCUSSION TITLE
build hang up: Collecting page data in Nextjs(appDir)

### DISCUSSION THREAD

**Discussion by y0zong** [vercel/community#1633] [Help]
### Summary

...an hour has passed
<img width="1369" alt="Screen Shot 2023-02-27 at 11 12 53" src="https://user-images.githubusercontent.com/86962713/221464858-a7181d53-592f-49e7-b391-8207ef40edcc.png">

**NO error**, **NO info**, **NO stop**, but I can success build in local. so what really happens during `Collecting page data`？

`nextjs` version v13.2.2(latest)

**Comment by swarnava** [comment]
Do you have a [minimal repository](https://github.com/evedriver/minimal-reproducable) which we can look into?
